lib/components/sidebarCard/sidebarCard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Card as AntCard } from 'antd';
+import styles from './sidebarCard.css';
+
+export const Card = (props) => {
+  return (
+    <AntCard
+      {...{
+        className: styles.wrap,
+        bordered: false,
+        ...props,
```

Comment:
Same here, we should limit the props to what our component will allow since this is not a general component, but one specifically made for the sidebar.

Reply:
fixed